django_irods/README.rst

```diff
@@ -0,0 +1,44 @@
+ga_irods - iRODS connector for Django with Celery
```

Comment:
@alvacouch @aphelionz We may want to add a summary sentence to indicate this django_irods app is copied from a django_irods submodule which was forked from the original django_irods repo, with a link to it, then we can add more specific and more relevant readme docs while removing not-so-relevant readme docs.

Reply:
I updated both README and CREDITS